vmm/src/config.rs

```diff
@@ -450,6 +486,9 @@ impl<'a> VmParams<'a> {
         let fs: Option<Vec<&str>> = args
             .get_many::<String>("fs")
             .map(|x| x.map(|y| y as &str).collect());
+        let vhost_user: Option<Vec<&str>> = args
+            .get_many::<String>("vhost-user-device")
```

Comment:
Weren't we going to call it user-vhost-device?  More consistent with user-device IMO.